public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1501 @@
+Ubuntu cloud image artifacts
+============================
+This document provides detailed information on various Ubuntu cloud image artifacts available on `cloud-images.ubuntu.com <http://cloud-images.ubuntu.com/>`_.
+
+Images
+------
+This section contains information on all of the Ubuntu cloud images available for download. These images are pre-configured and ready for deployment in cloud environments, supporting multiple architectures and configurations.
+
+Architectures supported
+~~~~~~~~~~~~~~~~~~~~~~~
+-  **amd64:** Standard 64-bit PC architecture.
+-  **arm64:** 64-bit ARM architecture.
+-  **armhf:** Hard-float 32-bit ARM architecture.
+-  **ppc64el:** 64-bit PowerPC little-endian architecture.
+-  **riscv64:** 64-bit RISC-V architecture.
+-  **s390x:** IBM System z (s390x) architecture.
+
+Initial ramdisk (initrd)
+~~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>-initrd-generic``
+   * - Example filename
+     - ``noble-server-cloudimg-amd70-initrd-generic``
+   * - Format description
+     - An initial ramdisk is a temporary root file system loaded into memory during the boot process to initialise the system before the real root file system is mounted.
+   * - Use cases
+     - Initrds are used to ensure the kernel can boot by loading necessary drivers and modules before the root filesystem is mounted. This allows the support of diverse hardware and virtual environments, making them essential for cloud instance startup. To facilitate this, initrds can include custom initialisation scripts and they offer modular kernel support.
+
+Linux kernel image
+~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>-vmlinuz-generic``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-vmlinuz-generic``
+   * - Format description
+     - The vmlinuz image contains a compressed image of the Linux kernel.
+   * - Use cases
+     - The Linux kernel is the core component of the operating system. It handles essential functions such as process management, memory management and system calls. Kernel images may be customised for specific hardware configurations, leading to offerings for each supported architecture.
+
+LXD tarball
+~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>.lxd.tar.xz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-lxd.tar.xz``
+   * - Format description
+     - Tar archive compressed with XZ (LZMA2), containing an image suitable for LXD container deployment.
+   * - Use cases
+     - These files are specifically formatted for LXD, a system container manager. They contain complete file system images that can be used to instantiate LXD containers. You can use ``.lxd.tar.xz`` files to create isolated environments with specific configurations and applications, ensuring consistent container deployments across LXD hosts.
```

Comment:
Negative. Those lxd tarballs only contain lxd metadata. the root filesystem is actually ``-root.tar.xz``.
You can use `.lxd.tar.xz` in conjunction with `-root.tar.xz` to spin containers.

Reply:
Thanks for the clarification